### PR TITLE
fix autoria request

### PR DESCRIPTION
**Pull request by R0ixy**
Now we use `compectation_Ids` to get `autoria_codes` and make request

### CODE REVIEW

File: backend/src/controllers/cars/cars.controller.ts [hunk 1]
```diff
@@ -10,12 +10,18 @@ import * as carsService from '@services/cars/cars.service';
...
 
 const carsSearchAutoria = async (
-  req: TypedRequestQuery<CarsSearchParams>,
+  req: TypedRequestQuery<{
+    complectationId: string[];
+    page: string;
+  }>,
   res: Response,
   next: NextFunction,
 ): Promise<void> => {
```
Comment (oleksandr-danylchenko) [inline backend/src/controllers/cars/cars.controller.ts]: I have a few questions: 
1. `complectationId` is singular, but its type is array :eyes:?
2. Why do we allow to pass multiple complectations at once? We will show autoria options for only one complication at a time

Comment (R0ixy) [reply backend/src/controllers/cars/cars.controller.ts]: On the page where we can choose complectations, we use checkbox so it looks like we can specify many complactations

Comment (oleksandr-danylchenko) [reply backend/src/controllers/cars/cars.controller.ts]: Nope, the checkbox was removed from the design and implementation plan. It was discussed in the meetings.

Also, on e-katalog the checkbox is responsible for adding an item to the comparison

**Review by oleksandr-danylchenko — APPROVED** [review]
Nice! Thanks :raised_hands:

**Pull request merged**